project.clj

```diff
@@ -46,7 +46,8 @@
                       [org.jolokia/jolokia-jvm                   "1.6.0" :classifier "agent"]
                       [clj-time                                  "0.15.1"]
                       [com.github.grinnbearit/freemarker-clj     "-SNAPSHOT"]
-                      [irresponsible/tentacles                   "0.6.2"]
+                      [irresponsible/tentacles                   "0.6.2" :exclusions [org.apache.ant/ant]]
+                      [org.apache.ant/ant                        "1.10.5"]
```

Comment:
A better solution here is to force an upgrade of [`pathetic` to v0.5.1](https://clojars.org/pathetic/versions/0.5.1).  

That version doesn't have a dependency on ClojureScript, which thereby also completely removes the dependency on ant (a vastly better solution).